docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,87 @@
+
+Once you've got the tool setup - you can benchmark with the most common transcoding `ABR ladder <https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_bench/transcodingOptions.json>`_ (``transcodingOptions.json``).
+
+For example to test for a range of concurrent sessions on a single Nvidia GPU with device id **'0'** you can use the following bash script:
+
+::
+
+    #!/bin/bash
+
+    for i in {1..20}
+    do
+        ./livepeer_bench -in bbb/source.m3u8 \
+        -transcodingOptions transcodingOptions.json \
+        -nvidia 0 \
+        -concurrentSessions $i |& grep "Took" >> bench.log
+    done
+
+To test on a CPU omit the ``-nvidia 0`` flag and change the loop's maximum from **20** to something lower like **5**.
+
+You will see the final output in a file called ``bench.log`` as following
+
+::
+
+    Took <X> seconds to transcode 30 segments of total duration 60s (1 concurrent sessions)
+    Took <X> seconds to transcode 30 segments of total duration 60s (2 concurrent sessions)
+    ...
+    Took <X> seconds to transcode 30 segments of total duration 60s (20 concurrent sessions)
+
+The goal here is to have the transcode time ``X`` remain within real-time, leaving about ~20% buffer room for network transit. Thus ``X <= 60s/1.20 = 50 seconds``.
+
+If your transcode time was quite fast even for the limit of **20** sessions in the above script, feel free to increase it to something higher. If you have multiple GPUs you can multiply whatever limit you calculate with a single GPU above, or pass all your devices in like ``-nvidia 0,1,2``.
+
+
+**2. Bandwidth Capacity**
+
+The most common transcoding `ABR ladder <https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_bench/transcodingOptions.json>`_ found on the network is (assuming source is ``1080p30fps``) - 
+
+
++------------------------+----------+
+| Resolution & Fps       | Bitrate  |
++========================+==========+
+| 1080p30fps             | 6000kbps |
++------------------------+----------+
+| 720p<source>fps        | 3000kbps |
++------------------------+----------+
+| 480p<source>fps        | 1600kbps |
++------------------------+----------+
+| 360p<source>fps        | 800kbps  |
++------------------------+----------+
+| 240p<source>fps        | 250kbps  |
++------------------------+----------+
+
+For a single stream you require 6000 kbps for fetching source rendition and 5600 kbps for sending the transcoded renditions. Thus you will roughly need -
+
+*Download Bandwidth* = **6 Mbps** Per Stream
+
+*Upload Bandwidth* = **5.6 Mbps** Per Stream
+
+Divide these from your network provider limits to get an idea of the number of streams you can handle. For example a typical broadband connection with upstream/downstream of 100 Mbps can serve ~16 streams reliably. You can probably stretch it by ~20% more as not all streams' segments would be processed at the same time.
```

Comment:
WDYT of linking to something like Ookla speedtest here ?

Reply:
Didn't want to recommend a particular speed test provider, but can for sure mention running one or even link to a google search for "speed test" wdyt?

Reply:
That's totally fine if we don't want to recommend one , I'm not sure of their affiliations either. I'm okay with adding a suggestion to use a speedtest to establish down/up speed as most non-datacenter connections will be asymetrical (e.g. my business connection is 300 down / 30 up )

Reply:
Yeah makes sense, added a quick google search link in the update

Reply:
We could link to this [bandwidth reference](https://livepeer.readthedocs.io/en/latest/reference/bandwidth_requirements.html).